newlib/libc/time/mktime.c

```diff
@@ -163,13 +171,12 @@ validate_structure (struct tm *tim_p)
     }
 }
 
-time_t 
-mktime (struct tm *tim_p)
+static time_t
+mktime_utc (struct tm *tim_p, long *days_p)
```

Comment:
I'd love to see a sized type for the 'long' here -- any reason this shouldn't be int32_t?